src/Products/SimpleProductEntity.php

```diff
@@ -7,7 +7,7 @@ class SimpleProductEntity implements \JsonSerializable
 {
     protected int $id;
     protected string $name;
-    protected float $price;
+    protected string $price;
```

Comment:
I would leave the price as a float so that you can sort / compare to other prices / convert the currency. But then you could convert it to a string from the getter instead.